test/client/js/libs/handlebars/ircmessageparser/parseStyle.js

```diff
@@ -48,6 +50,29 @@ describe("parseStyle", () => {
 		expect(actual).to.deep.equal(expected);
 	});
 
+	it("should parse strikethrough", () => {
```

Comment:
Can you add a couple more tests for this, like combining with italics, colors and stuff like that.

Reply:
Absolutely